src/components/Alert.js

```diff
@@ -1,26 +1,41 @@
 var React = require('react');
 var classNames = require('classnames');
+var utils = require('../utils');
 
 var ALERT_TYPES = [
 	'danger',
 	'error', // alias for danger
 	'info',
-	'primary',
 	'success',
-	'warning',
+	'warning'
 ];
 
 module.exports = React.createClass({
 	displayName: 'ElementalAlert',
 	propTypes: {
 		children: React.PropTypes.node.isRequired,
 		className: React.PropTypes.string,
-		type: React.PropTypes.oneOf(ALERT_TYPES).isRequired,
+		theme: utils.getThemeValidator({
+			classes: ALERT_TYPES
+		}),
+		type: React.PropTypes.oneOf(ALERT_TYPES).isRequired
+	},
+	getDefaultProps () {
+		return {
+			theme: {
+				classes: {
+					danger: 'Alert Alert--danger',
+					error: 'Alert Alert--danger',
```

Comment:
@MikeMcElroy if you checkout [line 7](https://github.com/elementalui/elemental/pull/150/files/81764bf408a0d4050428c9befb7032d2fcd7e4f5#diff-3523f9197339e201dc1c7b9ad9c65877R7) `error` is just an alias for `danger`.